src/main/kotlin/io/zerobase/smarttracing/resources/DevicesResource.kt

```diff
@@ -1,34 +1,70 @@
 package io.zerobase.smarttracing.resources
 
-import io.zerobase.smarttracing.models.IdWrapper
+import com.fasterxml.jackson.annotation.JsonProperty
+import io.zerobase.smarttracing.GraphDao
+import io.zerobase.smarttracing.models.*
 import javax.ws.rs.*
 import javax.ws.rs.core.MediaType
 
 //region Request Models
+data class CreateDeviceRequest(@JsonProperty("fingerprint") val fingerprint: Fingerprint)
+
+data class CreateCheckInRequest(
+        @JsonProperty("scannedId") val scannedId: ScannableId,
+        @JsonProperty("type") val type: ScanType,
+        @JsonProperty("location") val location: Location?
```

Comment:
the annotation is only required if the field name doesn't match the json. just remove them if they do

Reply:
Fixed